server/models.py

```diff
@@ -6,11 +6,19 @@
 
 class Lecturer(db.Model):
     id = db.Column(db.Integer, primary_key=True)
+
+    user_name = db.Column(db.String(200), unique=True, nullable=False)
+    password = db.Column(db.String(200), nullable=False)
+    active = db.Column(db.Boolean, default=True)
+    admin = db.Column(db.Boolean, default=False)
```

Comment:
Yes, I was thinking about admin panel. For now we don't use it, I can remove it and add if we will manage to create this panel.